submissions/Barasii/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,95 @@
+<! doctype html>
```

Comment:
space between `!` and `doctype` is not valid. This instruction will be recognized as bogus comment. See more info in [validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fbarasii.github.io%2FHTML-CSS-POPUP%2F)